lib/main_common.pm

```diff
@@ -1968,7 +1968,7 @@ sub load_x11_gnome {
 sub load_x11_other {
     if (check_var("DESKTOP", "gnome")) {
         loadtest "x11/brasero/brasero_launch";
-        loadtest "x11/gnomeapps/gnome_documents";
+        loadtest "x11/gnomeapps/gnome_documents" unless is_tumbleweed();
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check. How about `if is_sle('<=15-sp3') or is_leap('<=15.3');`?

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

Reply:
That would probably be if is_sle('<16') or is_leap('<16') then; it is unlikely that a package disappears in a future Service Pack

Reply:
ok, then `if is_sle('<16') or is_leap('<16')`